devenv/bootstrap.py

```diff
@@ -133,7 +133,16 @@ def main(coderoot: str, argv: Sequence[str] | None = None) -> ExitCode:
             )
 
         print("Installing sentry's brew dependencies...")
-        proc.run((f"{homebrew_bin}/brew", "bundle"), cwd=f"{coderoot}/sentry")
+        if CI:
+            # Installing everything from brew takes too much time,
+            # and chromedriver cask flakes occasionally. Really all we need to
+            # set up the devenv is colima. This is also required for arm64 macOS GHA runners.
+            # TODO: pin colima in sentry via vendored formula and install from that
```

Comment:
i plan on doing this soon - 0.6.3 is the good version

https://github.com/abiosoft/colima/issues/898